tests/conf/i18n/translatehelpervisitor.js

```diff
@@ -0,0 +1,71 @@
+const TranslateHelperVisitor = require('../../../conf/i18n/translatehelpervisitor');
+const Translator = require('../../../conf/i18n/translator');
+const Handlebars = require('handlebars');
+const _ = require('lodash');
+
+async function createTranslator () {
+  const locale = 'fr';
+  return Translator.create(locale, [], { [locale]: { translation: {
+    'Hello': 'Bonjour',
+    'result': 'résultat',
+    'result_plural': 'résultats'
+  } } });
+}
+
+describe('TranslateHelperVisitor usage', () => {
+  it('static translation', async () => {
+    const template = `{{ translate phrase='Hello' }}`;
+    const ast = Handlebars.parse(template);
+    const translator = await createTranslator();
```

Comment:
do we need to create the translator twice, or could this be done inside the describe block?

Reply:
I moved this to a beforeEach in the describe block